packages/wrangler/src/vectorize/query.ts

```diff
@@ -28,7 +28,7 @@ export function options(yargs: CommonYargsArgv) {
 				type: "array",
 				demandOption: true,
 				describe:
-					"Vector to query the Vectorize Index. Example: `--vector 1 2 3 0.5 1.25 6`",
+					"Vector to query the Vectorize Index. Example: `--vector 1 2 3 0.5 1.25 6`. To read from a json file that contains data in the format [1, 2, 3], use `--vector $(jq -r '.[]' data.json | xargs)`",
```

Comment:
this expects the user to have jq and xargs on their system. shouldn't we do this in wrangler itself?

Reply:
That's a good idea. I have created a task for this in our backlog. I have also updated this command to be more of a suggestion than an instruction. I hope that would be sufficient for this change.